src/libprojectM/PipelineMerger.cpp

```diff
@@ -5,6 +5,14 @@
 const double PipelineMerger::e(2.71828182845904523536);
 const double PipelineMerger::s(0.5);
 
+void PipelineMerger::ensureAlphaIsNotBlended(const Pipeline & a)
+{
+	for ( std::vector<RenderItem*>::const_iterator pos = a.drawables.begin(); pos != a.drawables.end(); ++pos )
```

Comment:
```suggestion
	for (auto &pos : a.drawables)
```
I think is equivalent?

Reply:
It is.  Up to you if you want to require C++11.

Reply:
I believe we do already and I don't see a reason not to. It's fine without this change though